xwiki-platform-core/pom.xml

```diff
@@ -321,6 +321,19 @@
                 </item>
               </differences>
             </revapi.differences>
+            <revapi.differences>
+              <justification>Not a breakage, new annotation on existing protected and now public method.</justification>
+              <criticality>allowed</criticality>
+              <ignore>true</ignore>
+              <differences>
+                <item>
+                  <code>java.annotation.added</code>
+                  <old>method java.lang.String com.xpn.xwiki.objects.classes.ListClass::getDisplayValue(java.lang.String, java.lang.String, java.util.Map&lt;java.lang.String, com.xpn.xwiki.objects.classes.ListItem&gt;, com.xpn.xwiki.XWikiContext) @ com.xpn.xwiki.objects.classes.StaticListClass</old>
+                  <new>method java.lang.String com.xpn.xwiki.objects.classes.ListClass::getDisplayValue(java.lang.String, java.lang.String, java.util.Map&lt;java.lang.String, com.xpn.xwiki.objects.classes.ListItem&gt;, com.xpn.xwiki.XWikiContext) @ com.xpn.xwiki.objects.classes.StaticListClass</new>
+                  <annotation>@org.xwiki.stability.Unstable</annotation>
```

Comment:
Weird that you only get a revapi problem about the annotation and not about the visibility that has changed. Did you try to build `xwiki-platform-oldcore-legacy` with your changes? It's possible that revapi is only executed there to also include legacy code in the API comparison.

Reply:
> It's possible that revapi is only executed there

Yes, it's only executed in legacy module.

Reply:
It is indeed breaking because I missed all the other child classes of `ListClass` as I didn't try building the legacy module. However, making the method public is not breakage. As discussed in the [chat](https://matrix.to/#/!ikPtGZaGWtyblizzlR:matrix.xwiki.com/$1637144122457QMuoU:matrix.xwiki.com?via=matrix.org&via=matrix.xwiki.com&via=nobis.cloud), I've opened https://github.com/xwiki/xwiki-commons/pull/151 to ignore the `@Unstable`-annotations in general. This makes all of these annotations here obsolete. Should I remove them here or should I still add all the necessary annotations for now such that this can be merged?

Reply:
> Should I remove them here

you should yes.